Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -1864,21 +1864,45 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
     ///   - bundle: The bundle containing the articles.
     private func synthesizeArticleOnlyRootPage(articles: inout [DocumentationContext.SemanticResult<Article>], bundle: DocumentationBundle) {
         let title = bundle.displayName
-        let metadataDirectiveMarkup = BlockDirective(name: "Metadata", children: [
-            BlockDirective(name: "TechnologyRoot", children: [])
-        ])
-        let metadata = Metadata(from: metadataDirectiveMarkup, for: bundle, in: self)
+        
+        // An inner helper function to register a new root node from an article
+        func registerAsNewRootNode(_ articleResult: SemanticResult<Article>) {
+            uncuratedArticles.removeValue(forKey: articleResult.topicGraphNode.reference)
+            let title = articleResult.source.deletingPathExtension().lastPathComponent
+            // Create a new root-looking reference
+            let reference = ResolvedTopicReference(
+                bundleIdentifier: bundle.identifier,
+                path: NodeURLGenerator.Path.documentation(path: title).stringValue,
+                sourceLanguages: [DocumentationContext.defaultLanguage(in: [])]
```

Comment:
Probably we should give `in:` a default value of `[]` or `nil` so this reads a bit better:

```
sourceLanguages: [DocumentationContext.defaultLanguage]
```

Reply:
Because this is only a private method I would prefer to not add a default value so that future callers have to think about what languages they pass. 

If there was a default value there's a slight risk that a future caller that has available language information uses the default value and gets the wrong result back which could lead to a subtle bug. Having to explicitly pass `nil` makes this a tiny bit more explicit than passing no value (possibly due to autocompletion).

I added an inline comment that provides additional context to why `nil` is passed here.